quasar/main.tf

```diff
@@ -294,6 +294,78 @@ resource "aws_db_parameter_group" "quasar-qa" {
   #  }
 }
 
+resource "aws_db_parameter_group" "quasar-qa-pg11" {
+  name   = "quasar-qa-pg11"
+  family = "postgres11"
+
+  # Required for DMS CDC Replication:
+  # https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Source.PostgreSQL.html#CHAP_Source.PostgreSQL.v10
```

Comment:
Just wondering this comment should point to [CHAP_Source.PostgreSQL.RDSPostgreSQL](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Source.PostgreSQL.html#CHAP_Source.PostgreSQL.RDSPostgreSQL) instead. This section holds the recommendation to use this parameter.

> option

ameter

Reply:
The original link pointed to the Postgres 10 link, so might be worth an update, but we're deprecating DMS soon, so we'll probably remove the code. leaving it up to Morgan.

Reply:
That 10.x link points to the documentation for version 10 and later, so we can leave that for the moment.